### PR TITLE
fix(package.json): fix lint-staged version

### PR DESCRIPTION
**Pull request by jukben**
Fix the version. The 4.2.3 is the latest one.

### CODE REVIEW

**Comment by jukben** [comment]
image

File: app/templates/_package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     <% if (jest) { %>"jest": "^21.0.2",<% } %>
     <% if (prettier) { %>"prettier": "^1.6.1",<% } %>
     <% if (precommit) { %> "husky": "^0.14.3",
-    "lint-staged": "^4.4.2",<% } %>
+    "lint-staged": "^4.2.3",<% } %>
```
Comment (krizzu) [inline app/templates/_package.json:35]: Consider upgrading to the latest version v8.1.5.

**Review by krizzu — CHANGES_REQUESTED** [review]
Looks great, nice work 👏 	

Left small comment to check out, let me know once you're done with it!

File: app/templates/_package.json (hunk 1 above)
Comment (jukben) [reply app/templates/_package.json:35]: XDDD

**Comment by jukben** [comment]
Sorry, my friends, I'm retired now.

**Pull request closed without merging**